build/get-webpack-config.js

```diff
@@ -182,10 +184,15 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
         target: runtime === 'server' ? 'node-bundled' : 'browser-legacy',
         specOnly: false,
       });
-
+  console.log('--runtime is: ' + runtime);
```

Comment:
😉

Reply:
Also is this new runtime still used now we are using the forked worker-loader?

Reply:
Had some old references to this runtime which were outdated and have since been removed. Everything should now point to the forked worker-loader, which is included inside of this repo now (worker-loader.js)